sirepo/package_data/static/js/activait.js

```diff
@@ -1388,7 +1418,6 @@ SIREPO.app.directive('imagePreview', function(appState, requestSender, panelStat
                     $scope.colBName = 'Images';
                 }
             }
-
             initFromResponse($scope.imageInfo);
         }
     };
```

Comment:
I opened the Oxford Pets example on a fresh sirepo database, and saw this error before it loaded the file normally:

```Application Error: TypeError: Cannot read properties of undefined (reading 'setCustomValidity'). ```
```
angular.js:15567 TypeError: Cannot read properties of undefined (reading 'setCustomValidity')
    at $scope.validateNumSelected (activait.js?20240328.184633:1161:19)
    at setModel (activait.js?20240328.184633:1103:24)
    at activait.js?20240328.184633:1175:17
    at m.$eval (angular.js:19393:16)
    at b (angular.js:19535:11)
    at t (angular.js:19793:11)
    at m.$digest (angular.js:19096:11)
    at m.$apply (angular.js:19500:13)
    at angular.js:21601:25
    at Tg.completeTask (angular.js:21231:7)
(anonymous) @ angular.js:15567
```